content/index.pl.md

```diff
@@ -0,0 +1,125 @@
+-   narzędzia do tworzenia multimediów
+-   witryny internetowe, które pozwalają użytkownikom dodawać treści, takie jak blogi, wiki, strony służące do udostępniania zdjęć, fora internetowe i portale społecznościowe
+-   inne rodzaje narzędzi wymienione w słownikowej definicji [narzędzi autorskich](https://www.w3.org/TR/ATAG20/#def-Authoring-Tool)
+
+ATAG i zasoby pomocnicze mają również zaspokoić potrzeby wielu różnych odbiorców, w tym decydentów, menedżerów i innych. Na przykład:
+
+-   Osoby szukające narzędzia autorskich, które są dostępne i tworzą dostępne treści, mogą użyć ATAG do oceny narzędzi autorskich.
+-   Osoby, które chcą zachęcić swojego dotychczasowego programistę narzędzi autorskich do poprawy dostępności w przyszłych wersjach, mogą skierować twórcę narzędzia autorskiego do ATAG.
+
+## Co jest w ATAG
+
+**ATAG** są podzielone na dwie główne części::
+
+-   Część A skupia się na dostępności samego narzędzia autorskiego.
+-   Część B skupia się na wspieraniu autorów w tworzeniu dostępnych treści przez narzędzie autorskie.
+
+[[**ATAG w skrócie**]](/standards-guidelines/atag/glance/) zawierają krótkie streszczenie zasad i wytycznych dla dostępności ujętych w ATAG.
+
+**ATAG** są zorganizowany warstwowo:
+
+-   **Zasady**  zapewniają najwyższy poziom organizacji wytycznych.
+-   **Wytyczne**  zapewniają ramy i cele dla kryteriów sukcesu.
+-   **Kryteria sukcesu to wymagania dostępności**, które są napisane jako sprawdzalne stwierdzenia na [trzech poziomach](https://www.w3.org/TR/ATAG20/#intro_understand_levels_conformance):
+    A, AA, AAA.
+
+[**ATAG 2.0**](https://www.w3.org/TR/ATAG20/) są _Rekomendacją W3C_, która jest standardem technicznym.
+
+[**Wdrażanie ATAG 2.0**](https://www.w3.org/TR/IMPLEMENTING-ATAG20/)
+ to pomocniczy dokument informacyjny, który pomaga czytelnikom zrozumieć i używać ATAG. Wdrażanie ATAG zawiera uzasadnienie dla każdej wytycznej. Dla każdego kryterium sukcesu zawiera wyjaśnienie intencji kryteriów sukcesu, przykłady i łącza do zasobów.
+
+## Narzędzie do raportowania ATAG
+
+[**Narzędzie raportowania ATAG**](https://www.w3.org/WAI/atag/report-tool/) pomaga oceniającym raportować wyniki oceny dostępności narzędzi autorskich. Prowadzi użytkownika przez wymagania ATAG, umożliwia rejestrowanie wyników oceny dla każdego wymagania i generuje raport zgodności narzędzia autorskiego z ATAG.
+
+## Tłumaczenia
+
+ATAG 2.0 są dostępne jako autoryzowane tłumaczenie W3C w języku chińskim, uproszczonym: [无障碍创作工具指南(ATAG) 2.0](https://www.w3.org/Translations/ATAG20-zh/).
+
+## Kto opracował ATAG
+
+ATAG zostały opracowane przez Authoring Tool Accessibility Guidelines Working Group ([ATAG WG](https://www.w3.org/WAI/AU/)), która jest częścią Inicjatywy na Rzecz Dostępności Internetowej ([WAI](https://www.w3.org/WAI/)) w Konsorcjum <span lang="en">World Wide Web</span> ([W3C](http://www.w3.org)).
```

Comment:
@zwiastunsw Would you please translate "Authoring Tool Accessibility Guidelines Working Group"?